yt/frontends/arepo/data_structures.py

```diff
@@ -73,6 +73,10 @@ def _get_uvals(self):
         handle.close()
         if missing:
             uvals = None
+        else:
```

Comment:
Is there a chance `"UnitLength_in_cm"` would not exist if `"UnitMass_in_g"` or `"UnitVelocity_in_cm_per_s"` does? Maybe it would be more robust with 

```suggestion
        if uvals["UnitLength_in_cm"]:
```

?

Reply:
Yes, I agree.

Reply:
Actually, this doesn't work, because the logic of the code demands that all three exist for the units to be specified at all.

Reply:
ok! then leaving it as is is good!